### PR TITLE
set selected flag on selected object option; resolves #1325

### LINKED ISSUE #1325: Gaze clicked: Selected object in the dropdown disappears after any paste operation in the graph
Open this to reproduce:
http://vizor.io/fthr/3d-model-selected/edit

In the root, there is a Gaze Clicked plugin, with the 3D model selected in the dropdown.
Copy-paste any of the subgraphs just for test, and observe how the "3D model" get unselected from the dropdown. This happens also when any preset is dropped from the patches list.

### PR DESCRIPTION
**Pull request by kschzt**

### CODE REVIEW

File: browser/scripts/abstractObjectGazePlugin.js [hunk 1]
```diff
@@ -182,8 +182,10 @@
 
 		findSelectableMeshNodes()
 		.map(function(selectable) {
+			var nodeRef = selectable.node.parent_graph.uid + '.' + selectable.node.uid
```
Comment (gmarinov) [inline browser/scripts/abstractObjectGazePlugin.js]: a plugin that knows about parent graphs.. is this something the node can return? looks reusable.

Comment (kschzt) [inline browser/scripts/abstractObjectGazePlugin.js]: the plugin needs to know when stuff changes in the graph, to update the selector

Comment (kschzt) [inline browser/scripts/abstractObjectGazePlugin.js]: I'll add a getGraph() method on Node

**Comment by gmarinov** [comment]
lgtm

**Pull request merged**